core/src/initializers/settings.ts

```diff
@@ -143,6 +143,14 @@ export class Plugins extends CLSInitializer {
           "The default country code Grouparoo will use to format phone numbers and display data",
         type: "string",
       },
+      {
+        key: "confirm-profiles-days",
+        title: "Confirm Profile Existence Days",
+        defaultValue: 7,
```

Comment:
👍 this is a good default.  We will check once a week for profiles we haven't imported in a while and delete them if they are removed from the source.

cc @andyjih

Reply:
@andyjih agrees that once a week is OK for now